### DISCUSSION TITLE
How do you import an ontology in Protege?

### DISCUSSION THREAD

**Discussion by JisooSeo** [JisooSeo/101-WG-Discussions#1] [Q&A]
How do you import an ontology in Protege?

**Comment by JisooSeo** [comment]
Protégé – “Active ontology” tab – “Ontology Imports” tab – “Direct imports” - import .owl file

**Reply by johnbeve** [reply]
Suggestion: Perhaps include screenshots alongside the answer description. Even better, you can embed a short video of you navigating on your local machine to illustrate.

**Reply by JisooSeo** [reply]
Thank you! That's a great idea!
